intelmq/bots/collectors/shadowserver/collector_reports_api.py

```diff
@@ -138,15 +138,18 @@ def process(self):
                 self.logger.debug('Processed file %r (fixed: %r) already.', filename, filename_fixed)
                 continue
             self.logger.debug('Processing file %r (fixed: %r).', filename, filename_fixed)
-            reportdata = self._report_download(item['id'])
-            report = self.new_report()
-            report.add('extra.file_name', filename_fixed)
-            report.add('raw', reportdata)
-            self.send_message(report)
-            self.cache.set(filename, 1)
-            self.logger.debug('Sent report: %r (fixed: %r, size: %.3g KiB).', filename, filename_fixed,
+            try:
+                reportdata = self._report_download(item['id'])
+                report = self.new_report()
+                report.add('extra.file_name', filename_fixed)
+                report.add('raw', reportdata)
+                self.send_message(report)
+                self.cache.set(filename, 1)
+                self.logger.debug('Sent report: %r (fixed: %r, size: %.3g KiB).', filename, filename_fixed,
                               len(reportdata) / 1024)  # TODO: Replace by a generic size-conversion function
-            reports_downloaded += 1
+                reports_downloaded += 1
+            except:
+                self.logger.error("Data download ERROR: %r, %r!", item['file'], item['id'])
```

Comment:
That hides the actual error message, using `self.logger.exception` the exception is still displayed. Or, adding the string representation or a description of the exception of the error message ("Timeout on data download..."), that would be much shorter.